vulnz/src/docker/scripts/mirror.sh

```diff
@@ -33,5 +42,12 @@ if [ -n "${DEBUG}" ]; then
   DEBUG_ARG="--debug"
 fi
 
-exec java $JAVA_OPT -jar /usr/local/bin/vulnz cve $DELAY_ARG $DEBUG_ARG $MAX_RETRY_ARG $MAX_RECORDS_PER_PAGE_ARG --cache --directory /usr/local/apache2/htdocs
+function remove_lockfile() {
+  rm -f $LOCKFILE
+  exit 0
+}
+trap remove_lockfile SIGHUP SIGINT SIGQUIT SIGABRT SIGALRM SIGTERM SIGTSTP
+
+java $JAVA_OPT -jar /usr/local/bin/vulnz cve $DELAY_ARG $DEBUG_ARG $MAX_RETRY_ARG $MAX_RECORDS_PER_PAGE_ARG $CONTINUE_ARG --cache --directory /usr/local/apache2/htdocs
```

Comment:
Can we look for an exit code and if it failed for a single year - try kicking it off again automatically?

Reply:
i like the idea generally, but IMHO we should try this in a separate PR, what do you think?

Reply:
I'm fine with a separate PR. We just need to consider outputting the correct exit code that we can look for here.

Reply:
Specifically, I'm referencing https://github.com/jeremylong/Open-Vulnerability-Project/pull/259/files#diff-cd4981d15e7c01b10b719bd871fee87886e1b1e67a4dc314caf7d3416d47c77fR295-R302

Reply:
@jeremylong i implemented exit codes within the `processRequest` call, 100 for "partial years" and 101 for "modified file failed"

But to be honest, i'am not sure how this one is exposed in the `TimedCommand#call` - seems like it is used as the computation result.

So if and who is actually returning a exit code i did not find - maybe you can help with that.

This would be the first step, that we have proper exit code. The devops part of "restart if that" i would care about in an additional PR

Reply:
I'll follow up with another PR for the exit code.